packages/dscc-scripts/src/viz/util.ts

```diff
@@ -69,24 +70,69 @@ export const validateBuildValues = (args: VizArgs): BuildValues => {
   };
 };
 
-const validateJSON = (jsonStr: string, schema: any, fn: string) => {
+const friendifyError = (error: Ajv.ErrorObject): string =>
+  `The value at: ${error.dataPath} is invalid. ${error.message}.`;
+
+const unique = <T>(ts: T[]): T[] => [...new Set(ts)];
+
+const validateWithSchema = (o: object, schema: object): boolean => {
   const ajv = new Ajv({allErrors: true});
-  const jsonObject = JSON.parse(jsonStr);
-  const testJson = ajv.compile(schema);
-  const isValid = testJson(jsonObject);
-  // testJson.errors
-  if (isValid === false) {
-    throw invalidVizJSON(fn, testJson.errors);
+  const configValidator = ajv.compile(schema);
+  const isValidConfig = configValidator(o);
```

Comment:
Styling nit: I'm generally not a fan of single character variable names.

Reply:
ack. I think I'll keep it because I can't call it object, and other names that come to mind aren't much better.